src/oculusviewer.cpp

```diff
@@ -61,7 +61,7 @@ void OculusViewer::configure() {
                      m_device->projectionMatrix(OculusDevice::Eye::RIGHT),
                      m_device->viewMatrix(OculusDevice::Eye::RIGHT),
                      true);
-  m_viewer->getSlave(1)._updateSlaveCallback =
+  m_viewer->getSlave(m_viewer->getNumSlaves()-1)._updateSlaveCallback =
```

Comment:
You have `getSlave(m_viewer->getNumSlaves()-1)` for both conditions. That cannot be correct.

Reply:
It's correct because each call to getSlave() is done right after an addSlave(), which increases the number of slaves. This way you always retrieve the last slave just added.